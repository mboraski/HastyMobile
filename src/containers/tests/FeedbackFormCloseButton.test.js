// TODO: Remove when tests implemented
test.only('real recognize real', () => {
    expect('real').toBe('real');
});
// import 'react-native';
// import React from 'react';
// import { shallow } from 'enzyme';
//
// import { FeedbackFormCloseButton } from '../FeedbackFormCloseButton';
//
// describe('FeedbackFormCloseButton', () => {
//     it('should render correctly', () => {
//         const props = {};
//         const render = shallow(<FeedbackFormCloseButton {...props} />);
//         expect(render).toMatchSnapshot();
//     });
//     it('should render correctly form visible', () => {
//         const props = { feedbackFormVisible: true };
//         const render = shallow(<FeedbackFormCloseButton {...props} />);
//         expect(render).toMatchSnapshot();
//     });
// });
