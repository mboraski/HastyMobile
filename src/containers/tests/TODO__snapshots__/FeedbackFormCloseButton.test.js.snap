// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedbackFormCloseButton should render correctly 1`] = `""`;

exports[`FeedbackFormCloseButton should render correctly form visible 1`] = `<Connect(CloseButton) />`;
